### PR TITLE
add gfs

### PR DESCRIPTION
**Pull request by liasece**
##### Twitter Link:
https://twitter.com/StarCrazyGame

##### List of audit links if any:
N/A

##### Website Link:
https://starcrazy.com

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://github.com/iotexproject/iotex-token-metadata/blob/master/images/io1t585efypl4e9ex7xks2upnjm8saawfk0v0s5sj.png

##### Current TVL:
58.23 k

##### Chain:
IoTeX

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
https://www.coingecko.com/en/coins/game-fantasy-token

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
https://coinmarketcap.com/currencies/game-fantasy-token/

##### Short Description (to be shown on DefiLlama):
StarCrazy is the NFT and blockchain Play-To-Earn game. Collect, fuse, mine, and earn. GFS is its main game currency.

##### Token address and ticker if any:
0x5d0f4ca481fd725c9bc6b415c0ce5b3c3bd726cf

##### Category (Yield/Dexes/Lending/Minting/Assets/Insurance/Options/Indexes/Staking) *Please choose only one:
Yield

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
N/A

##### forkedFrom (Does your project originate from another project):
N/A

##### methodology (what is being counted as tvl, how is tvl being calculated):
Coins pledged in liquid mining pools.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/gfs exports TVL: 
        
 
 
    iotex                     58.29 k
    
    total                    58.29 k

File: projects/gfs/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+const tvl = async (timestamp, block, chainBlocks) => {
+  let chain = "iotex";
+  block = await getBlock(timestamp, chain, chainBlocks);
+  let balances = { "iotex": 0 };
+
+  const [balanceOfTokenListedInLP] = await Promise.all([
+    sdk.api.erc20.balanceOf({
+      target: wiotx,
+      owner: liquidityPool,
```
Comment (g1nt0ki) [inline projects/gfs/index.js:20]: Sorry, but you cant add entire liquidity pool as tvl, you can only count LP staked by the users on your staking contract and as 'pool2' not under 'tvl'

**Comment by llamatester** [comment]
The adapter at projects/gfs exports TVL: 
        
 
 
    iotex                     0
    iotex-pool2               50.95 k
    pool2                     50.95 k
    
    total                    0

File: projects/gfs/index.js (hunk 1 above)
Comment (liasece) [reply projects/gfs/index.js]: Oh, i have been updated the code, review again pls

**Pull request merged**